sources/packages/backend/apps/api/src/services/institution/institution.service.ts

```diff
@@ -694,6 +695,37 @@ export class InstitutionService extends RecordDataModelService<Institution> {
       .getOne();
   }
 
+  /**
+   * Check if institution is private
+   * @param institutionId is the institution id.
+   * @returns boolean true if institution is private, else false.
+   */
+  async checkIfInstitutionIsPrivate(institutionId: number): Promise<boolean> {
```

Comment:
Just a minor change that I missed in the previous review, a method like this usually will be named as `isPrivateInstitution`, IMO.